.github/workflows/test.yml

```diff
@@ -30,15 +30,23 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: "14.x"
+      - name: Download the latest stable version of Meilisearch
+        run: |
+          curl -L https://install.meilisearch.com | sh
+          chmod +x meilisearch
+      - name: Run Meilisearch
+        run: |
+          ./meilisearch --master-key=masterKey --no-analytics true &
       - name: Install dependencies
-        run: yarn --dev && yarn --cwd ./playgrounds/vue
-      - name: Run Browser tests
+        run: yarn --dev && yarn --cwd ./tests/env/react
```

Comment:
Should we create a matrix with react, vue, angular and vanilla javascript? Just in case to prevent possible breaking between them.

Reply:
It would take a very very long time for the tests to run. I thought maybe run them once a week in a scheduled job but not on every PR. What do you think ?